sql/server.go

```diff
@@ -151,8 +153,17 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 type parameters []driver.Datum
 
 // Arg implements the Args interface
-func (p parameters) Arg(i int) (parser.Datum, bool) {
-	if i < 1 || i > len(p) {
+func (p parameters) Arg(name string) (parser.Datum, bool) {
+	if !unicode.IsDigit(rune(name[0])) {
```

Comment:
My thinking was that `rune(name[0])` would give the first byte as a `rune` which would only contain ASCII digits. I agree that is a bit subtle. I'll change this to something more explicit in a subsequent PR.